src/test/java/com/tagtraum/perf/gcviewer/imp/TestDataReaderShenandoah.java

```diff
@@ -29,46 +25,130 @@ private DataReader getDataReader(GCResource gcResource) throws IOException {
 
     @Test
     public void parseBasicEvent() throws Exception {
-        TestLogHandler handler = new TestLogHandler();
-        handler.setLevel(Level.WARNING);
-        GCResource gcResource = new GcResourceFile("SampleShenandoahBasic.txt");
-        gcResource.getLogger().addHandler(handler);
+        GCModel model = getGCModelFromLogFile("SampleShenandoahBasic.txt");
+        assertThat("size", model.size(), is(5));
+        assertThat("amount of STW GC pause types", model.getGcEventPauses().size(), is(2));
+        assertThat("amount of STW Full GC pause types", model.getFullGcEventPauses().size(), is(0));
+        assertThat("amount of concurrent pause types", model.getConcurrentEventPauses().size(), is(3));
+        assertThat("total pause time", model.getPause().getSum(), is(0.001318));
```

Comment:
I think, tests for "double" values should usually be made using Matchers.closeTo() (examples can be found in TestDataReaderSun1_8_0G1). The reason is, that the representation of double values is not exact and can lead to tests failing randomly.